test/lib/pgBackRestTest/Module/Mock/MockAllTest.pm

```diff
@@ -63,13 +63,13 @@ sub run
 
     foreach my $rhRun
     (
-        {vm => VM1, remote => false, s3 =>  true, encrypt => false, delta =>  true, compress =>  GZ},
+        {vm => VM1, remote => false, s3 =>  true, encrypt => false, delta =>  true, compress => LZ4},
         {vm => VM1, remote =>  true, s3 => false, encrypt =>  true, delta => false, compress =>  GZ},
         {vm => VM2, remote => false, s3 => false, encrypt =>  true, delta =>  true, compress =>  GZ},
         {vm => VM2, remote =>  true, s3 =>  true, encrypt => false, delta => false, compress =>  GZ},
         {vm => VM3, remote => false, s3 => false, encrypt => false, delta =>  true, compress =>  GZ},
-        {vm => VM3, remote =>  true, s3 =>  true, encrypt =>  true, delta => false, compress =>  GZ},
-        {vm => VM4, remote => false, s3 => false, encrypt => false, delta => false, compress =>  GZ},
+        {vm => VM3, remote =>  true, s3 =>  true, encrypt =>  true, delta => false, compress => LZ4},
+        {vm => VM4, remote => false, s3 => false, encrypt => false, delta => false, compress => LZ4},
         {vm => VM4, remote =>  true, s3 =>  true, encrypt =>  true, delta =>  true, compress =>  GZ},
     )
```

Comment:
1) Why is compress not passed to begin() like it is in other mock tests?
2) Since strCompressType => NONE is NOT being passed to setup() then how is it defaulting to NONE (it is passed in MockStanza etc)

Reply:
This was fixed during review of the cmp-int branch, but that branch was never merged here.